spec.md

```diff
@@ -648,34 +648,67 @@ Additionally, a field may optionally be declared with a *flipped* orientation.
 {word: UInt<32>, valid: UInt<1>, flip ready: UInt<1>}
 ```
 
-In a connection between circuit components with bundle types, the data carried by the flipped fields flow in the opposite direction as the data carried by the non-flipped fields.
 
-As an example, consider a module output port declared with the following type:
+#### Alignment: Flipped vs Aligned
+
+A subcomponent's alignment is a relative property: a subcomponent is aligned/flipped relative to another subcomponent of the same component or type.
+Hence, a subcomponent is flipped/aligned *with respect to (w.r.t)* another subcomponent of that type (parent, sibling, child etc.).
+
+The following example of a non-nested bundle `Parent` is used to list all of the alignment relationships between members of `p`.
 
-``` firrtl
-output a: {word: UInt<32>, valid: UInt<1>, flip ready: UInt<1>}
+```
+wire p: { alignedChild: UInt<32>, flip flippedChild: UInt<32> }
 ```
 
-In a connection to the `a`{.firrtl} port, the data carried by the `word`{.firrtl} and `valid`{.firrtl} sub-fields will flow out of the module, while data carried by the `ready`{.firrtl} sub-field will flow into the module.
-More details about how the bundle field orientation affects connections are explained in [@sec:connects].
+First, every subcomponent/component is always aligned with themselves:
 
-As in the case of vector types, a bundle field may be declared with any type, including other aggregate types.
+- `p` is aligned w.r.t `p`
+- `p.alignedChild` is aligned w.r.t `p.alignedChild`
+- `p.flippedChild` is aligned w.r.t `p.flippedChild`
 
-``` firrtl
-{real: {word: UInt<32>, valid: UInt<1>, flip ready: UInt<1>},
- imag: {word: UInt<32>, valid: UInt<1>, flip ready: UInt<1>}}
+Next, all parent/child relationships:
+Because the `flippedChild` field is `Flipped`, it changes its aligment relative to its parent.
 
-```
+- `p` is aligned w.r.t `p.alignedChild`
+- `p` is flipped w.r.t `p.flippedChild`
```

Comment:
this is sort of weirdly written. I know it is a symmetrical relationship, but wouldn't it be more intuitive to write it as

`p.aligned child is aligned w.r.t. p`